stdlib/public/core/String.swift

```diff
@@ -110,6 +112,18 @@ public protocol StringProtocol
   ) rethrows -> Result
 }
 
+extension StringProtocol {
+  //@available(swift, deprecated: 3.2, obsoleted: 4.0, message: "Please use the StringProtocol itself")
+  //public var characters: Self { return self }
```

Comment:
Why are these commented?

Reply:
Looks like this was merged with these commented out.  We're seeing the effects of this in Swift 3.2.1.  Was this intentional ?